.github/workflows/check-migrations.yml

```diff
@@ -0,0 +1,35 @@
+name: Check Migrations on Paseo
+
+on:
+  pull_request_target:
+    types:
+      - labeled
+  workflow_dispatch:
+    inputs:
+      runtime-package:
+        description: 'Runtime package to check migrations for'
+        required: true
+        default: 'frequency-runtime'
+      runtime-uri:
+        description: 'URI of the runtime to check migrations for'
+        required: true
+        default: 'wss://0.rpc.testnet.amplica.io:443'
+    
+
+jobs:
+  check-migrations:
+    if: contains(github.event.pull_request.labels.*.name, 'change/storage-migration')
```

Comment:
Filter out other labels that may trigger this

Reply:
Doesn't this also filter out the `workflow_dispatch`?

Reply:
You are absolutely correct, good catch, I will fix it